src/CMakeLists.txt

```diff
@@ -33,17 +24,14 @@ foreach(lib objtreelite objtreelite_runtime objtreelite_common)
     $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
     $<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/include>
     $<INSTALL_INTERFACE:$<INSTALL_PREFIX>/include>)
+  target_link_libraries(${lib} PUBLIC Threads::Threads)
```

Comment:
Do we need to link `${CMAKE_THREAD_LIBS_INIT}`?

Reply:
Not necessary, since we are already linking `Threads::Threads` target.

Reply:
I couldn't remember why I added that to xgboost, maybe for mingw ...  We can think about it when it comes up again.